src/doc/tarpl/coercions.md

```diff
@@ -0,0 +1,72 @@
+% Coercions
+
+Types can implicitly be coerced to change in certain contexts. These changes are
+generally just *weakening* of types, largely focused around pointers and lifetimes.
+They mostly exist to make Rust "just work" in more cases, and are largely harmless.
+
+Here's all the kinds of coercion:
+
+
+Coercion is allowed between the following types:
+
+* Subtyping: `T` to `U` if `T` is a [subtype](lifetimes.html#subtyping-and-variance)
```

Comment:
It might be worth pulling subtyping out separately - because it can be done anywhere, not just in coercions, this is more like a reflexivity property of coercions.